apps/tutorial-servo-can-control/TutorialServo.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
```

Comment:
check your indenting on this file, depending on your code editor you can probably select a setting that can format your code every time you save.  If that's not an option make sure you go through and manually indent.

Reply:
There's a section in our repo readme called "Clang-Format" that walks you through setting up the formatter we use. Please go through it and use it to format the training files. Our formatting style uses an indentation of 2 spaces so you'll know you've formatted your file properly if the indentation is 2 spaces.
![image](https://user-images.githubusercontent.com/36703188/120818692-d64e1980-c520-11eb-9a6e-1d8ce4370058.png)